k8s.gcr.io/images/k8s-staging-cluster-api-azure/OWNERS

```diff
@@ -10,6 +10,12 @@ approvers:
 reviewers:
```

Comment:
>  Should always mirror the maintainers/reviewers in https://sigs.k8s.io/cluster-api-provider-azure/OWNERS_ALIASES

https://github.com/kubernetes/k8s.io/pull/4594/files#diff-fd84133b7e59974ea0a0a37206686a1ba7b6e8c6301ca2dc67745f79802e6fa9R2

So let's add `sonasingh46` for now since he's missing (https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/OWNERS_ALIASES#L26) but everyone else who is an "active contributor" should first become a reviewer of CAPZ to follow k8s governance.

/hold

Reply:
`alexeldeib` should also be removed from approvers since he was removed from https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/OWNERS_ALIASES

Reply:
Good call on the 1st point: https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2984